### PR TITLE
Add transactions and notifications to core docs

### PR DESCRIPTION
**Pull request by Szymx95**

### CODE REVIEW

File: docs/source/core.rst [hunk 1]
```diff
@@ -353,6 +411,30 @@ Returns allowance (tokens left to use by spender) for given tokenOwner - spender
     {allowance && <p>Remaining allowance: {formatUnits(allowance, 18)} tokens</p>}
   )
 
+.. _useTransactions:
+
+useTransactions
+===============
+
+``useTransactions`` hook returns a list ``transactions`` of transactions sent from useDApp.
```
Comment (marekkirejczyk) [inline docs/source/core.rst:419]: list of what transactions? Perhaps all transactions that were triggered with useContractFunction or useSendTransaction?

File: docs/source/guide.rst [hunk 2]
```diff
@@ -379,78 +379,79 @@ Withdraw function needs amount of ether to withdraw as a input argument.
...
-
-Take a look at example usage below:
+To access history of transaction sent by useDApp, use :ref:`useTransactions` hook.
 
 .. code-block:: javascript
 
   const { transactions } = useTransactions()
 
+``transactions`` is an array so you can use ``transactions.map(...)`` to display all of 
+transactions.
```
Comment (marekkirejczyk) [inline docs/source/guide.rst:389]: I would add links to core docs

**Review by marekkirejczyk — CHANGES_REQUESTED** [review]
see comments

File: docs/source/guide.rst (hunk 2 above)
Comment (Szymx95) [reply docs/source/guide.rst:389]: There is a link to core docs on line 382, or should it be a different link ?

**Pull request merged**